mesh_map/CMakeLists.txt

```diff
@@ -18,7 +18,10 @@ foreach(dep IN LISTS dependencies)
   find_package(${dep} REQUIRED)
 endforeach()
 
-find_package(Boost REQUIRED COMPONENTS system)
+find_package(LVR2 REQUIRED)
+find_package(MPI)
+# this is nowhere used
+# find_package(Boost REQUIRED COMPONENTS system) 
```

Comment:
True, let's just delete it
```suggestion
```

Reply:
I just noticed that your changes were not committed. I will create another PR for that.